### DISCUSSION TITLE
Ability to select all fields (including custom) in the columns for Requestable Assets

### DISCUSSION THREAD

**Discussion by mw-christensen** [snipe/snipe-it#11411] [Ideas]
Maybe I am missing something simple but currently when an employee goes to "Requestable Assets", only a limited number of columns are available to view. I would like it so that all columns (or columns an admin specifies) are viewable. The only reason employees would be selecting "Requestable Assets" is because they need something with certain specs (such as processor type, RAM, GPU, etc).

Thanks.

**Comment by technigogo** [comment]
I support this feature request.
